src/tooltip.js

```diff
@@ -250,6 +250,8 @@
                         .styleTween('transform', function (d) {
                             return translateInterpolator;
                         })
+                        // not using tween for webkit touch device
+                        .style('-webkit-transform', new_translate)
```

Comment:
Are you sure this doesn't remove the tooltip move animation for non-mobile webkit browsers?